### DISCUSSION TITLE
How to edit tags page

### DISCUSSION THREAD

**Discussion by kodumulo** [adityatelange/hugo-PaperMod#394] [Q&A]
I'm using this theme very useful
Thanks for working on you on this theme!
Is there any way to add a separate title meeting room icon on the tag page?

**Comment by adityatelange** [comment]
You can add `content/tags/index.md` and define title as you'd do for any other post.
That title should reflect into Tags pages title instead of the default one.

**Reply by BytePrince** [reply]
I have made it like what you said:  add `content/tags/index.md` ,and the ` index.md` contains :
---
title: "About23444433"
layout: "tags"
---

but when I finish it,the layout of the tag page had chaged to the normal post's.

Just like this : https://pic.imgdb.cn/item/60963acdd1a9ae528fe69d2d.jpg

**Reply by kdkasad** [reply]
I have no idea if this will help but try renaming `index.md` to `_index.md` (with an underscore).

**Comment by BytePrince** [comment]
Thank you very much!
It works!

Kian Kasad ***@***.***> 于2021年5月9日周日 上午10:43写道：
